src/api/spec/controllers/webui/badge_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'webmock/rspec'
+require 'rails_helper'
+RSpec.describe Webui::BadgeController, vcr: true do
```

Comment:
```suggestion

RSpec.describe Webui::BadgeController, type: :controller do
```
No need to require `webmock/rspec` or use `vcr`. It's a good practice to set the type of the spec, in this case to :controller.

Reply:
Sorry, the comment was right, the suggestion was wrong. It should have removed the `require 'webmock/rspec'`. You fixed it. Thanks! :+1: